packages/next/src/server/lib/incremental-cache/fetch-cache.ts

```diff
@@ -253,22 +233,19 @@ export default class FetchCache implements CacheHandler {
           throw new Error(`invalid response from cache ${res.status}`)
         }
 
-        const json: IncrementalCacheValue = await res.json()
-        const parsed = zCachedFetchValue.safeParse(json)
+        const cached = await res.json()
 
-        if (!parsed.success) {
-          this.debug && console.log({ json })
-          throw new Error('invalid cache value')
+        if (!cached || cached.kind !== 'FETCH') {
+          this.debug && console.log({ cached })
+          throw new Error(`invalid cache value`)
         }
 
-        const { data: cached } = parsed
-
         // if new tags were specified, merge those tags to the existing tags
         if (cached.kind === 'FETCH') {
           cached.tags ??= []
           for (const tag of tags ?? []) {
             if (!cached.tags.includes(tag)) {
-              cached.tags.push(tag)
+              cached.tag.push(tag)
```

Comment:
```suggestion
              cached.tags.push(tag)
```

This was part of the typo fix.